packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeEthForm/ConfirmStakeEthModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Button, Checkbox, H2, Icon } from '@trezor/components';
+import { spacingsPx } from '@trezor/theme';
+import { Modal, Translation, TrezorLink } from 'src/components/suite';
+import { useDispatch } from 'src/hooks/suite';
+import { openModal } from 'src/actions/suite/modalActions';
+
+const StyledModal = styled(Modal)`
+    width: 500px;
+    text-align: left;
+`;
+
+const VStack = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${spacingsPx.lg};
+    margin-top: ${spacingsPx.xl};
+`;
+
+const Flex = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${spacingsPx.md};
+`;
+
+const Divider = styled.div`
```

Comment:
Please use `Divider` component from `trezor/components`

https://github.com/trezor/trezor-suite/pull/11252

Reply:
This is a fresh change, which wasn't present at the time of creating the PR. According to your contribution rules, I shouldn't rebase when a review is in progress. I guess I can do it in the last turn, when all other comments are resolved.

Reply:
https://github.com/everstake/trezor-suite/commit/521430a37d9c6545ea5f951f98a9c03d2549060a

Done. Though, there’s not much sense in it in this context, since I still need to modify it heavily.

This divider is used inside a modal. The modal background has the same color as this new `Divider` in the dark theme, making it invisible. So I need a custom color here. Also, according to the design I need some changes of the width as well. Since you don’t use the className approach anymore, I ended up with a wrapper for a divider.